File: Week1~Python/Mukemmel-Sayi.py
# Mükemmel sayı
# Kodunuzu bu dosya içine yazınız
"""
Kodunuzun düzgün çalışması için
input metodunun içindeki metni boş bırakın
input('')
şeklinde...
"""
sayi = int(input(''))
top = 0
for i in range(1, sayi):
    if(sayi % i == 0):
        top += i
        
if(top == sayi):
    print("mükemmel")
else:
    print("mükemmel değil")

File: Week4~Ruby/Mukemmel-Sayi.rb
# Taş-kağıt-makas
# Kodunuzu bu dosya içine yazınız
=begin
Kodunuzun düzgün çalışması için
input metodunun içindeki metni boş bırakın
gets.chomp
yeterli...
=end

sayi = gets.chomp.to_i

top = 0


for i in 1...sayi
    if sayi % i == 0
        top += i
    end
end

if top == sayi
    puts "mükemmel"
else
    puts "mükemmel değil"
end